doc/source/conf.py

```diff
@@ -106,6 +106,8 @@ def setup(app):
     "sphinx.ext.intersphinx",
     "sphinx.ext.coverage",
     "sphinx_copybutton",
+    "sphinx_design",
```

Comment:
Add this as a dependency in toml file. the latest version is 0.5.0( FYI)

Reply:
Done